src/components/course/course-header/CourseRunCards.jsx

```diff
@@ -37,14 +42,16 @@ const CourseRunCards = () => {
           );
         }
         return (
-          <CourseRunCard.Deprecated
+          <DeprecatedCourseRunCard
```

Comment:
I wonder if its worth backlogging the tech-debt to update this card away from deprecation status?

Reply:
Unfortunately, it'll be around in a deprecated state for awhile, as long as we need to support legacy enterprise offers and coupon codes, and while subscriptions are not in the EMET system. It feels a bit premature to backlog a ticket to get rid of it when there's no immediate timeline on when we could action on it.